src/shared/SellerProfile/Body.js

```diff
@@ -120,7 +120,7 @@ const Body = (props) => {
       </Row>
 
       <Row title="Documents" show={!isEmpty(documents)}>
-          {Object.keys(documents).map((key, val) => {
+          {!isEmpty(documents) && Object.keys(documents).map((key, val) => {
```

Comment:
Yes, without it empty documents was still crashing render. I'm guessing it's still evaluated but passed to Row which then decides if it's not rendered?